### ISSUE TITLE
Feature Request: Show Keyboard hits

### ISSUE THREAD

**Issue by X3MBoy** [MaartenBaert/ssr#867]
There are lots of programs that do this outside SSR, but it will be easier to have it inside. And example could be "[screenkey](https://github.com/wavexx/screenkey)"

**Comment by MaartenBaert** [comment]
Duplicate of #259. I don't really see the added value of integrating this in SSR, what's wrong with using an external program for this?

**Comment by boban-dj** [comment]
I agree, key-mon is an excellent complement tool for SSR:[https://code.google.com/archive/p/key-mon/downloads](url) and here for archlinux users: [https://aur.archlinux.org/packages/key-mon/](url)
![Screenshot_2021-05-05_14-48-43](https://user-images.githubusercontent.com/5364226/117143462-2a929c00-adb1-11eb-95c5-f2575b050dc2.png)
